wiki/Tournaments/JHC/JHC_2024/en.md

```diff
@@ -75,6 +331,274 @@ The **JHC 2024** (***Jack House Cup 2024***) is a team–based osu!mania 4K tour
 - Speed Jack
   1. [Hommarju - Crazy Jackpot (\[GB\]V1do) \[Stage 6: JACKpot\]](https://osu.ppy.sh/beatmapsets/2118126#mania/4448636)
 
+## Match results
+
+The full match results are also available in the [master sheet](https://docs.google.com/spreadsheets/d/1Qm2BfIc2LFbf59-2RBkqnOde95y2WgoyO_XRNSrvNa4/).
+
+### Grand Finals
+
+Saturday, 9 March 2024:
+
+| # | Team Red |  |  | Team Blue | Match link | VOD link |
+| :-: | --: | :-: | :-: | :-- | :-: | :-: |
+| 77 | GangWonDuo | **7** | 5 | hsod | [#1](https://osu.ppy.sh/community/matches/113001768) | [#1](https://www.bilibili.com/video/BV12Z42187nP/) |
+
+Sunday, 10 March 2024:
+
+| # | Team Red |  |  | Team Blue | Match link | VOD link |
+| :-: | --: | :-: | :-: | :-- | :-: | :-: |
+| 78 | wehehe | **7** | 0 | GangWonDuo | [#1](https://osu.ppy.sh/community/matches/113019292) | [#1](https://www.bilibili.com/video/BV1Nr421p76T/) |
+
+### Finals
+
+Saturday, 2 March 2024:
+
+| # | Team Red |  |  | Team Blue | Match link | VOD link |
+| :-: | --: | :-: | :-: | :-- | :-: | :-: |
+| 74 | SV Player | 0 | **6** | hsod | [#1](https://osu.ppy.sh/community/matches/112902834) | [#1](https://www.bilibili.com/video/BV1WH4y1j76j/) |
+| 76 | GangWonDuo | 3 | **6** | wehehe | [#1](https://osu.ppy.sh/community/matches/112902144) | [#1](https://www.bilibili.com/video/BV1zj421U7hi/) |
+
+Sunday, 3 March 2024:
+
+| # | Team Red |  |  | Team Blue | Match link | VOD link |
+| :-: | --: | :-: | :-: | :-- | :-: | :-: |
+| 73 | garfield exterminators if fortnite | **6** | FF | hawk | [#1](https://osu.ppy.sh/community/matches/112914432) | [#1](https://www.bilibili.com/video/BV1dz421D79L/) |
```

Comment:
on the wiki, forfeits are notated like this

```suggestion
| 73 | garfield exterminators if fortnite | **0** | -1 | hawk | [win by default](https://osu.ppy.sh/community/matches/112914432) | [#1](https://www.bilibili.com/video/BV1dz421D79L/) |
```

see https://github.com/ppy/osu-wiki/blob/master/wiki/Tournaments/TEMPLATE.md#match-results

Reply:
Is *italic* needed for "win by default"?

Reply:
ah yeah that's how it's in other articles